### ISSUE TITLE
"|Error| Invalid command" while trying to load project

### ISSUE THREAD

**Issue by ferbeb** [radare/radare2#8287]
Version: 
Docker
radare2 1.7.0-git 1 @ linux-x86-64 git.1.7.0-git
commit: 4dd45b73e96c02fd8e24c8064e947843ff9bd918 build: 2017-08-23__14:55:44

1. run gcc on "int main(){return 0;}"
2. open r2; analyze & seek to main function
3. CC what a nice main function
4. Ps test; exit and save.
5. open r2; Po test; confirm closing current session

Yields the following error message: `|ERROR| Invalid command '[00] 0x00000000 ---- va=0x00000000 sz=0x0000 vsz=0x0000 ' (0x5b)`. Nothing seems to be loaded from the project.

On Windows version radare2 1.7.0-git 15824 @ windows-x86-32 git.1.6.0-88-gb52cc8285
commit: b52cc8285bcf55b604423e1e9131b052b3411e00 build: 2017-07-21__18:12:23 the same on a different small executable seems to work, the comment loads, although with a long list of warnings:

```
Missing " in (e http.uproot = C:\Users\User\AppData\Local\Temp\").Unknown calling convention '(null)'
See afCl for available types
Unknown calling convention '(null)'
See afCl for available types
Unknown calling convention '(null)'
See afCl for available types
Unknown calling convention '(null)'
See afCl for available types
(...)
```

I hope this can be of assistance and that there is no error on my part. Thanks!

On a side note: I'm too stupid to compile for Windows - tried Meson, MinGW and Cygwin - would a docker for builds be something useful that could be considered in the future?

**Comment by radare** [comment]
Regression in git because of siol. Known issue already warned several times in twitter

**Comment by radare** [comment]
Fixed in 52d5b1fec1177c936068bb7e5a58566ed2e059f6
